ts/twitter/src/app/(protected)/home/_components/timeline/timeline-item-frame.tsx

```diff
@@ -10,6 +12,8 @@ import {
 import { Tooltip } from "@/lib/components/ui/tooltip";
 import type { Post, QuoteRepost } from "@/lib/models/post";
 import { Avatar, Box, Flex, HStack, IconButton, Text } from "@chakra-ui/react";
+import * as React from "react";
```

Comment:
```suggestion
import { useState } from 'react';
```
Both the X-clone development guidelines and the official documentation recommend this approach, but is there a specific reason why you chose to use the `import * as React from "react"` syntax instead?
Ref: https://react.dev/reference/react/useState

Reply:
I have no reason, just my lack of knowledge.